gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1256,12 +1396,10 @@ def main(args):
         # NOTE: The following step is not yet implemented for joint and should not
         # be reviewed for it
         if args.export_vcf:
-            if data_type == "joint":
-                raise ValueError("Joint data type is not yet supported for VCF export.")
             if contig and test:
                 raise ValueError(
-                    "Test argument cannot be used with contig argument as test filters"
-                    " to chr20, X, and Y."
+                    "Test argument cannot be used with contig argument as test uses the"
```

Comment:
But do we still need it here since we have it before all the if.args.*?

Reply:
Ah youre right, you can remove this